zipkin-lens/src/translations/zh-cn/messages.json

```diff
@@ -1,6 +1,7 @@
 {
   "Address": "",
```

Comment:
@jxc928 @jxc928 @uckyk @hanahmily can some of you help us finish this translation file? It is missing many text